src/Core/Renderer/DefaultCommandsCategoriesRenderer.cs

```diff
@@ -6,36 +6,33 @@
 //-----------------------------------------------------------------------------
 
 using WheelMUD.Server;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using WheelMUD.Utilities;
 
 namespace WheelMUD.Core
 {
-    using System;
-    using System.Collections.Generic;
-    using System.Linq;
-    using System.Text;
-
     [RendererExports.CommandsCategories(0)]
     public class DefaultCommandsCategoriesRenderer : RendererDefinitions.CommandsCategories
     {
         public override string Render(TerminalOptions terminalOptions, IEnumerable<Command> commands)
         {
             // Build a list of categories for the commands available to this player.
-            var sb = new StringBuilder();
-            sb.AppendLine("Please specify a command category:");
+            var ab = new AnsiBuilder();
+            ab.AppendLine("Please specify a command category:");
 
             foreach (CommandCategory category in Enum.GetValues(typeof(CommandCategory)))
             {
-                if (category != CommandCategory.None)
+                if (category == CommandCategory.None) continue;
```

Comment:
nit: We could also LINQ exclude this one from the foreach, if that feels cleaner than `continue` flow?

Reply:
already done.